src/extension.ts

```diff
@@ -64,11 +65,13 @@ export function activate(context: vscode.ExtensionContext) {
 
   CircleViewerProvider.register(context);
 
+  RelationViewerProvider.register(context);
+
   // returning backend registration function that will be called by backend extensions
   return backendRegistrationApi();
 }
 
 /* istanbul ignore next */
 export function deactivate() {
   // TODO do cleanup
-}
+}
```

Comment:
A newline please 😄

Reply:
Ooh, I'm sorry. I'll apply it right away.